stable/artifactory-ha/ReverseProxyConfiguration.md

```diff
@@ -10,33 +10,81 @@ the required configuration snippet which you can then download and install direc
 **Note**: Nginx image distributed with Artifactory helm chart is custom image managed and maintained by JFrog.  
   
 ####  Features of Artifactory Nginx
-*   Provides default configuration with self signed SSL certificate.
-*   Auto update reverse proxy configuration by pulling configuration generated by Reverse Proxy Configuration Generator from Artifactory using [REST API](https://www.jfrog.com/confluence/display/RTF/Configuring+a+Reverse+Proxy#ConfiguringaReverseProxy-RESTAPI)
+*   Provides default configuration with self signed SSL certificate generated on each helm install/upgrade.
 *   Persist configuration and SSL certificate in `/var/opt/jfrog/nginx` directory
   
-#### Steps to use Reverse Proxy Configuration Generator to configuration for reverse proxy in nginx (Artifactory Pro/Enterprise).  
-*   Deploy Artifactory using helm chart with Nginx enabled.
-*   Go to Artifactory Admin -> HTTP Settings. 
-    Example values: 
-    ```
-    Docker Access Method: SubDomain
-    Server Provider: Nginx
-    Internal Hostname: $ARTIFACTORY_SERVICE_NAME (Get Artifactory Service Name by running `kubectl get svc` command)
-    Public Server Name: $DOMAIN_NAME
-    SSL Key Path: /var/opt/jfrog/nginx/ssl/tls.key (If SSL Cert is provided via Secret)
-    SSL Certificate Path: /var/opt/jfrog/nginx/ssl/tls.crt (If SSL Cert is provided via Secret)
-    ```
-*   Provide appropriate values and save configuration.
-*   Once configuration is saved Nginx will automatically fetch reverse proxy configuration snippet from Artifactory and apply it immediately.
-  
+#### Changing the default Artifactory nginx conf
+Use a values.yaml file for changing the value of nginx.mainConf or nginx.artifactoryConf
+These configuration will be mounted to the nginx container using a configmap.
+For example:
+1. Create a values file `nginx-values.yaml` with the following values:
+```yaml
+nginx:
+  mainConf: |
+    ## add HA entries when ha is configure
+    ## Replace server names with Artifactory service names (primary and member service names)
+    upstream artifactory {
```

Comment:
Our current setup does not use the `upstream`. Nginx is aware of a single service that can include members only or primary + members in the pool.
This is controlled by key `artifactory.service.pool`. I'd like to keep it like this (unless you see a problem with it).

Reply:
yeah sure. I just thought I would keep it as it was before. Changed